src/Types/VoidType.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace ILIAS\Types;
+
+
+class VoidType implements Type {
```

Comment:
Do I understand correctly that this is different from `null` in the regard that it does not contain any value?